src/components/Hoverable/index.js

```diff
@@ -159,13 +159,19 @@ class Hoverable extends Component {
                 if (_.isFunction(child.props.onMouseEnter)) {
                     child.props.onMouseEnter(el);
                 }
+                if (_.isFunction(this.props.onMouseEnter)) {
+                    this.props.onMouseEnter(el);
+                }
```

Comment:
Move this to be the first callback (same for onMouseLeave)

Reply:
Done.

Reply:
This is still not the first, it needs to goes before setIsHovered